extra-packages.repos

```diff
@@ -0,0 +1,8 @@
+  sensor_component/external/tamagawa_imu_driver:
```

Comment:
```suggestion
repositories:
  sensor_component/external/tamagawa_imu_driver:
```

Reply:
This is necessary for vcs import to work: https://github.com/autowarefoundation/autoware/actions/runs/13536213847/job/37828286048?pr=5801